resources/public/api/conf/P1.0/application.raml

```diff
@@ -32,4 +32,14 @@ uses:
         description: Return hello world
         body:
           application/json:
-            example: "\"Hello World\""
\ No newline at end of file
+            example: "\"Hello World\""
+
+
+/contact-details:
+  get:
+    displayName: Contact Details
+    description: An endpoint that currently returns a NOT_IMPLEMENTED exception
+    is: [headers.acceptHeader]
+    responses:
+      200:
```

Comment:
Should be a 500 not a 200 

Also keep to the same description please for consistency. 👍 

https://github.com/hmrc/individuals-benefits-and-credits-api/pull/28/files#diff-db7c56de2f9a53f9fdbfb21ff8ce4667R44

Reply:
Done